recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -75,6 +92,20 @@ CleanupManager.addCleanupHandler(() => PreferenceExperiments.stopAllObservers())
 
 this.PreferenceExperiments = {
   /**
+   * Set the default preference value for active experiments that use the
+   * default preference branch.
+   */
+  async init() {
+    const store = await ensureStorage();
+    for (const experiment of Object.values(store.data)) {
+      const {expired, preferenceBranchType, preferenceName, preferenceValue} = experiment;
+      if (!expired && preferenceBranchType === "default") {
```

Comment:
It's super edge case-y: If we're doing a user pref experiment (and the pref doesn't have a default value already), and the experiment ends, the default will retain the experimental value for the rest of the session. This check prevents setting the default when unnecessary. Although the reason I added it in the first place was more that it made sense only to set the pref when necessary, rather than because I thought that edge case mattered.

I'm fine with removing it, although I don't see the harm in keeping it.